src/controllers/index.ts

```diff
@@ -29,7 +29,7 @@ router.use('/deposits', depositsRouter);
 router.use('/metrics', metricsRouter);
 router.use('/account', accountRouter);
 router.use('/widgets', widgetsRouter);
-router.use('/gas_station', gasStationRouter);
+router.use('/relay', relayHubRouter);
```

Comment:
Maybe consider to also keep the old route intact for a transition period..

Reply:
No need! No load, no problem🤠